### DISCUSSION TITLE
[프로그래머의 길, 멘토에게 묻다.] 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#35] [Q&A] [answered]
책의 지은이의 글을 읽고 든 생각입니다..!  

개인적인 생각으로 마무리할 수 있지만 멘토님의 생각이 궁금해서 글을 남깁니다..  

---

> 알지도 못하며 그 사실도 모르는 자, 바보로다 - 그를 멀리하라!  
> 알지 못하나 그 사실을 아는 자, 못 배운 자로다 - 그를 가르치라!  
> 알고 있으나 그 사실을 모르는 자, 잠든 자로다 - 그를 깨우라!  
> 알고 있으며 그 사실을 아는 자, 깨우친 자로다 - 그를 따르라!  

지은이의 글 첫 번째 글귀이다.  

여기서 `알지 못하나 그 사실을 아는자, 못배운 자`는 날 의미하는 것 과 같고 `알고 있으며 그 사실을 아는 자, 깨우친 자`는 멘토님을 말씀하시는 것 같다.  

내 시점에선 깨우친 자는 누군지 보이기 때문에 그를 따를 것이고 멘토님 시선에선 마찬가지로 못배운 자는 내가 되니 나를 가르친 것이다.  

항상 궁금했던 점은 멘토님은 목적없이 나에게 귀한 시간을 내주시면서 왜 선의를 배푸는 걸까?  

감사하는 마음과 꼭 보답하겠다는 마음은 항상 가지고 있고, 닮아가고 싶은 롤 모델이기도 하다.  

물론 배품을 좋아하셔서 봉사활동의 일환이라고 생각하실 수 있다.  

*(지식의 나눔을 정말 좋아하시는 것 같다.. 지식의 나눔이라는 추상적인 것 보다 낚시하는 방법에 대해서 많이 길을 잡아주신다. 똑바로 서있을 수 있도록 자세를 교정해주시는 느낌)*  

지금까지 크게 생각해본적 없이 이런 멘토링을 받았는데 위의 글귀를 읽고 다시 생각해보게 된 것 같다.  

멘토링이란 `경험과 지식이 많은 사람이 스승 역할을 하여 지도와 조언으로 그 대상자의 실력과 잠재력을 향상시키는 것 또는 그러한 체계`라고 한다.  

아직 책을 다 읽지 않았고 추측에 불과하지만 함께 자라기, 멘토링, 컨퍼런스등 이런 활동을 지속하시는 이유는 가르치면서 멘토님도 성장을 목적에 둔게 아닌가라는 생각이 든다.  

이러한 생각을 지속하다 보니 이게 사실이 아니라도 멘토와 멘티의 관계에 있어서 선순환은 존재해야 한다는 생각 든다.(받기만 하는 관계 x)  

지금까지 멘토링을 너무 생각없이 받은게 아닌가라는 생각도 들고 좀 더 다른 시선이나 내 생각을 이야기 나누며 다른 맛?을 필요로 하실까? 라는 고민이 생겼다.  

생각을 정리해보니 멘토님이 내게 바라는 것은 성장일 것이고 이런 관계에 대한 책임감도 조금 달라진 것 갇다.  

정리해보면서 든 생각이지만 멘토님은 `호밀밭 파수꾼`의 주인공의 꿈과 똑같다.  

> 몇천 명의 아이들이 있을 뿐 주위에 어른이라곤 나밖에 아무도 없어.  
> 나는 아득한 낭떠러지 옆에 서 있는 거야.  
> 내가 하는 일은 누구든지 낭떠러지에서 떨어질 것 같으면 얼른 가서 붙잡아 주는 거지.  
> 애들이란 달릴 때는 저희가 어디로 달리고 있는지 모르잖아?

---

내용은 책을 읽고 정리를 마크다운 형식으로 저장하다가 다 정리가 되면 블로그에 올리는 방식을 사용하려고 해서 글 문체가 이렇습니다.  

아직 책을 다 읽지 않아서 이에 대한 답이 뒤에 존재할 수 있다는 생각도 해봤습니다  

그래도 멘토님의 생각이 평소에도 궁금해서 글을 남깁니다.

**Comment by jongfeel** [comment, marked as answer]
아주 좋은 내용으로 정리해 주셨네요.
그리고 제 칭찬이 많이 있어서 감사할 따름입니다.

우선 제가 멘토링을 하는 이유는 여러 가지가 있는데 얘기해 주신 것에도 포함되어 있기는 합니다.

- 큰 그림: 내가 조금 먼저 갔던 길을 알려주는 길잡이 역할
- 사회적 측면: 인간은 사회적인 동물(존재) 라는 걸 잘 알고 있기에 큰 그림에서 그 사회에 기여를 하고 싶어하는 마음 (봉사일 수도 있을 듯 합니다)
- 기술적 측면: 그 사회 안에서 소프트웨어를 만든다는 것의 개념을 이해시켜 주고, 팀으로 혹은 팀이 아니더라도 협력해서 만들어 가는 방법 전달

인 것 같습니다.

그리고 저와 같이 얘기하는 것에 있어서 많은 부담은 안가지셔도 됩니다.
나중에 좋은 인맥으로 간접적인 도움 같은 걸 주실 수 있다면 그걸로도 충분하고
직접적으로 뭔가 보답해 주셔도 충분히 고마울 것 같기도 합니다.

크게는 사회에 그리고 또 이 길을 걸어오려는 분들에게 좋은 모습을 보여주시는게
저한테 보답해 주는 것 보다 더 좋을 것 같아요.
